### ISSUE TITLE
Use which method in startup.cs?

### ISSUE THREAD

**Issue by mia-li** [AzureADQuickStarts/AppModelv2-WebApp-OpenIDConnect-DotNet#49]
**I see using this method configure AAD auth in readme.**
![image](https://github.com/AzureADQuickStarts/AppModelv2-WebApp-OpenIDConnect-DotNet/assets/34880005/bca635e7-05dd-4bd9-b606-8ccf809bb60e)

**but in the real startup.cs it using another method:**
![image](https://github.com/AzureADQuickStarts/AppModelv2-WebApp-OpenIDConnect-DotNet/assets/34880005/f13cff68-f188-4c48-85ad-d632885ac10e)

**I'm wondering which method should I use? the later one?**

**Comment by jmprieur** [comment]
Yes, @mia-li : the one in the code. We need to update the README.md
